polaris-react/src/components/TextField/TextField.scss

```
@@ -202,8 +203,9 @@ $spinner-icon-size: 12px;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   left: 0;
   background-color: var(--p-surface);
-  border: var(--p-border-width-1) solid var(--p-border-subdued);
-  border-top-color: var(--p-border-shadow);
+  border: var(--p-border-width-1) solid var(--p-border-shadow);
+  // stylelint-disable-next-line -- hard coded to address accessbility issue https://github.com/Shopify/polaris/issues/7838
+  border-top-color: #898f94;
```

See comment: https://github.com/Shopify/polaris/pull/7893#pullrequestreview-1216267807

```suggestion
  // stylelint-disable-next-line -- hard coded to address accessbility issue https://github.com/Shopify/polaris/issues/7838
  border: var(--p-border-width-1) solid #898f94;
```